x-pack/legacy/plugins/index_management/server/routes/api/indices/register_freeze_route.ts

```diff
@@ -4,10 +4,16 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { Router, RouterRouteHandler } from '../../../../../../server/lib/create_router';
 
+interface ReqPayload {
+  indices: string[];
+}
+
+const handler: RouterRouteHandler = async (request, callWithRequest, h) => {
```

Comment:
@joshdover is there a plan for migration of `../../../../../../server/lib/create_router`? Specifically looking at `h` 😄 , but also the other values passed in.

Reply:
Ok, I see the plan is to move over to `http.route` NP service, right?

Reply:
That's my understanding